examples/unicorn.yml

```diff
@@ -0,0 +1,18 @@
+# Unicorn process example with RAILS_ENV variable
+
+processes:
+  - name: unicorn
+    config:
+      pid_file: tmp/pids/unicorn.pid
+      start_command: bundle exec unicorn -Dc config/unicorn/production.rb
```

Comment:
@ahorner I think the ENV reference should either be a separate example or a README update.